back/engines/free/email_campaigns/spec/fixtures/locales/mailers.en.yml

```diff
@@ -14,8 +14,17 @@ en:
       "initiative_assigned_to_you": Assignment of a proposal to me
       "initiative_marked_as_spam": Proposal spam report
       "initiative_published": Publication of my proposal
+      "internal_comment_on_idea_assigned_to_you": Internal comment on input assigned to me
```

Comment:
Do you know why we need this file? It just seems to copy `back/engines/free/email_campaigns/config/locales/en.yml`. Could we get rid of it?

Reply:
No. I was hoping you wouldn't ask! 😆 I'll ask in back-end slack channel.

Reply:
I deleted the file, and only 2 specs failed. I suggest I log a ticket for it's removal, and add the details there.

Reply:
Ticket created: [CL-3883](https://citizenlab.atlassian.net/browse/CL-3883)

[Asked in Slack](https://citizenlabco.slack.com/archives/C016C2EHURY/p1687949221569529) if anyone thinks this unwise.

Reply:
Ticket deleted, as Sebi explained ([Slack thread](https://citizenlabco.slack.com/archives/C016C2EHURY/p1687953415665319?thread_ts=1687949221.569529&cid=C016C2EHURY)) it why it's unwise to delete this file.